test/unit/lifecycle/properties.js

```diff
@@ -199,14 +199,27 @@ describe('lifecycle/property', function () {
         expect(elem.test).to.equal('something');
       });
 
-      it('context and arguments', function () {
+      it('context and arguments', function (done) {
```

Comment:
This test wasn't fleshed out enough, but I did add more parts to it to test the new params.